src/main/java/knightminer/inspirations/tools/InspirationsTools.java

```diff
@@ -76,172 +77,213 @@
 	public static Item northCompass;
 	public static Item barometer;
 	public static Item photometer;
-	public static Item waypointCompass;
+	public static ArrowItem redstoneArrow;
+
+	// The "undyed" compass is White.
+	public static ItemWaypointCompass[] waypointCompasses = new ItemWaypointCompass[16];
 
 	// tool materials
-	public static ToolMaterial bone;
-	public static ToolMaterial blaze;
-	public static ToolMaterial wither;
+	public static IItemTier bone = new IItemTier() {
```

Comment:
Assuming I keep the crooks (which is pretty unlikely at this point), I'd rather make a class that implements this interface and do this through the constructor, implementing a whole class inline is awkward.